adal/src/main/java/com/microsoft/aad/adal/PackageHelper.java

```diff
@@ -43,6 +43,8 @@
  */
 class PackageHelper {
     private static final String TAG = "CallerInfo";
+    private static final String PKG_COMPANYPORTAL = "com.microsoft.windowsintune.companyportal";
+    private static final String PKG_AZURE_AUTHENTICATOR = "com.azure.authenticator";
```

Comment:
Same above. Please check com.microsoft.aad.adal.AuthenticationConstants.Broker#AZURE_AUTHENTICATOR_APP_PACKAGE_NAME.

Reply:
Fixed in bbd79a83d57d0f5da497301a97306165c1f0fab4